### PR TITLE
build: :sparkles: Introduce typescript

### PR DESCRIPTION
**Pull request by sechaba-mache**

### CODE REVIEW

**Review by kphutiyagae — APPROVED** [review]

**Review by MihlaliMgwebi — CHANGES_REQUESTED** [review]
Add dist folder to your gitignore.
 
Including the "dist" folder in the repository can cause a number of issues such as bloating the size of the repository, making it slower to clone and download, and making it more difficult to manage changes and version control.

**Comment by sechaba-mache** [comment]
> Add dist folder to your gitignore.
> 
> Including the "dist" folder in the repository can cause a number of issues such as bloating the size of the repository, making it slower to clone and download, and making it more difficult to manage changes and version control.

Done, added build and .DS_Store as well

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
-node_modules
\ No newline at end of file
+node_modules
+dist
```
Comment (MihlaliMgwebi) [inline .gitignore:2]: Great

**Review by MihlaliMgwebi — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (sechaba-mache) [reply .gitignore:2]: 👍🏽

**Pull request merged**